Projects/Claims/Sources/Views/SubmitClaim/Emergency/SubmitClaimEmergencyScreen.swift

```diff
@@ -0,0 +1,164 @@
+import SwiftUI
+import hCore
+import hCoreUI
+
+struct SubmitClaimEmergencyScreen: View {
+    @State var selectedFields: [String] = []
+
+    var body: some View {
+        hForm {
+            VStack(spacing: 8) {
+                hSection {
+                    InfoCard(text: L10n.submitClaimEmergencyInfoLabel, type: .attention)
+                }
+                ClaimEmergencyContactCard(
+                    icon: hCoreUIAssets.hedvigBigLogo,
+                    label: L10n.submitClaimEmergencyGlobalAssistanceLabel,
+                    buttonText: L10n.submitClaimGlobalAssistanceCallLabel("+45 38 48 94 61"),
+                    cardTitle: L10n.submitClaimEmergencyGlobalAssistanceTitle,
+                    footnote: L10n.submitClaimGlobalAssistanceFootnote
+                )
+
+                hSection {
+                    VStack(alignment: .leading, spacing: 8) {
+                        hText(L10n.submitClaimEmergencyInsuranceCoverTitle)
+                        hText(L10n.submitClaimEmergencyInsuranceCoverLabel)
+                            .foregroundColor(hTextColorNew.secondary)
+                    }
+                }
+                .padding(.top, 16)
+                .sectionContainerStyle(.transparent)
+
+                VStack(spacing: 4) {
+                    infoExpandableView(
+                        title: L10n.submitClaimWhatCostTitle,
+                        text: L10n.submitClaimGlassDamageWhatCostLabel
+                    )
+                    infoExpandableView(
+                        title: L10n.submitClaimHospitalTitle,
+                        text: L10n.submitClaimGlassDamageWhatCostLabel
+                    )
+                    infoExpandableView(
+                        title: L10n.submitClaimRebookTitle,
+                        text: L10n.submitClaimGlassDamageWorkshopLabel
+                    )
+
+                    infoExpandableView(
+                        title: L10n.changeAddressQa,
+                        text: L10n.submitClaimGlassDamageWorkshopLabel
+                    )
+                }
+                .padding(.top, 16)
+                SupportView()
+                    .padding(.vertical, 56)
+            }
+            .padding(.top, 8)
+
+        }
+    }
+
+    func infoExpandableView(title: String, text: String) -> some View {
```

Comment:
Can we extract this to separated view so we can reuse it in both glass damage and emergency?